spec/requests/uploads_controller_spec.rb

```diff
@@ -409,7 +409,7 @@ def upload_file(file, folder = "images")
           sign_in(user)
           get upload.short_path
 
-          expected_max_age = S3Helper::DOWNLOAD_URL_EXPIRES_AFTER_SECONDS - UploadsController::SECURE_REDIRECT_GRACE_SECONDS
+          expected_max_age = SiteSetting.secure_media_cache_seconds - UploadsController::SECURE_REDIRECT_GRACE_SECONDS
```

Comment:
Now that this is a variable, should we run this validation in a site setting validator?